com.unity.netcode.gameobjects/CHANGELOG.md

```diff
@@ -34,6 +34,7 @@ Additional documentation and release notes are available at [Multiplayer Documen
 - Fixed issue where a client could throw an exception if abruptly disconnected from a network session with one or more spawned `NetworkObject`(s). (#2510)
 - Fixed issue where invalid endpoint addresses were not being detected and returning false from NGO UnityTransport. (#2496)
 - Fixed some errors that could occur if a connection is lost and the loss is detected when attempting to write to the socket. (#2495)
+- Fixed a failing UTP test that was failing when you install Unity Transport package 2.0.0 or newer.
```

Comment:
I don't think this is needed... We usually don't include test changes in our changelog/release notes.